.dockerignore

```diff
@@ -56,31 +56,19 @@
 /rest_api/sawtooth_rest_api/protobuf/
 /rest_api/**/__pycache__/
 
-/sdk/python/build/
-/sdk/python/sawtooth_sdk/protobuf/
-/sdk/python/**/__pycache__/
 /sdk/javascript/node_modules
```

Comment:
Should all reference of SDK be removed from sawtooth-core? Now that no SDK is with sawtooth-core and all are in their own repositories.

Reply:
I'll clean up the leftover stuff in a different PR.